venom/rpc/reflect/reflect.py

```diff
@@ -1,17 +1,26 @@
 from typing import Type, Set
 
 from venom import Message
-from venom.rpc import Method
+from venom.rpc.method import Method
 from venom.rpc import Service
 
 
 class Reflect(object):
-    services: Set[Service]
-    methods = Set[Method]
-    messages: Set[Type[Message]]
+    services = Set[Type[Service]]
```

Comment:
`Set[Type[Service]]` is the correct type annotation, but the use of a `=`, assigning the type hint, is incorrect here. 

Variable annotations use `:`: https://docs.python.org/3.6/whatsnew/3.6.html#whatsnew36-pep526